playground/pages/without.vue

```diff
@@ -0,0 +1,36 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+const show = ref(false)
+const CreditCardError = ref()
+const CreditCardToken = ref()
+</script>
+
+<template>
+    <h1>Load Mollie script lazily</h1>
+    <div>On this page the scripts load lazily, if the button is clicked.</div>
+    <div>To test this, the option <i>isIncludedGlobally: false</i> has to be set in nuxt config.</div>
+
+    <p>
+        <button @click="show = !show">{{show ? 'Hide' : 'Show'}} payment components</button>
+        <p v-if="show">
```

Comment:
The linter complains here `The template root requires exactly one element`

Reply:
See other comment :)